spec/requests/post_spec.rb

```diff
@@ -1,20 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
+  user = User.create(name: 'John', posts_counter: 30, photo: 'https://randomuser.me/api/portraits/men/70.jpg',
+                     bio: 'Teacher from Nigeria.')
```

Comment:
- [ ] hi when running your tests I get an error saying `unknown attribute 'posts_counter' for User.`. I suppose this is because you have `post_counter` and not `posts_counter`

Reply:
Thanks bro @m05e5